### ISSUE TITLE
redundantObjc: @objc on nested types still needed with @objcMembers

### ISSUE THREAD

**Issue by ChristopherRogers** [nicklockwood/SwiftFormat#384]
The following enum fails to be emitted to Obj-C if you remove its `@objc` annotation. The "redundantObjc" rule currently removes it as of 0.40.1.

```swift
@objcMembers
class MyClass: NSObject {
    @objc enum MyEnum: Int {
        case a
    }
}
```

**Comment by nicklockwood** [comment]
@ChristopherRogers thanks for reporting. I'll investigate and fix it asap. In the meantime, you can use a `// swiftformat:disable redundantObjc" directive to disable the rule inside that file.

**Comment by nicklockwood** [comment]
@ChristopherRogers fixed in 0.40.2.

**Comment by ChristopherRogers** [comment]
That was fast! Thanks!
